spec/raven/instance_spec.cr

```diff
@@ -16,25 +17,15 @@ private class InstanceTest < Raven::Instance
   end
 end
 
-private class LoggerTest < Raven::Logger
-  getter infos = [] of String
-
-  def info(message, *args)
-    super.tap do
-      @infos << message
-    end
-  end
-end
-
-def build_configuration
+def build_instance_configuration
   Raven::Configuration.new.tap do |config|
     config.dsn = "dummy://12345:67890@sentry.localdomain:3000/sentry/42"
-    config.logger = LoggerTest.new(nil)
+    config.logger = Raven::Logger.for(Raven::Logger::PROGNAME)
```

Comment:
This should use `Log::MemoryBackend`.

Reply:
That's done. Although, I don't fully understand why. Would you be so kind as to give me some pointers?

Reply:
Thanks! It's because we don't want to write to `STDOUT` during spec run.